src/main/java/com/sparrowwallet/sparrow/io/satochip/SatochipParser.java

```diff
@@ -0,0 +1,155 @@
+package com.sparrowwallet.sparrow.io.satochip;
+
+import com.sparrowwallet.drongo.protocol.Sha256Hash;
+import com.sparrowwallet.drongo.crypto.ECKey;
+import com.sparrowwallet.drongo.crypto.ECDSASignature;
+import com.sparrowwallet.drongo.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigInteger;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Base64;
+
+public class SatochipParser{
```

Comment:
why not put the class final with a private constructor and then set methods as statics to use them in that way?